### PR TITLE
README Documentation Additions

### LINKED ISSUE #39: README Examples Addition
Hello, 

Before I begin, I want to say great work on SwayOSD. I found this project a few days ago, and it is a nice addition to my Sway setup.

I want suggest adding this software's custom value usage to the README.

Currently, the README shows examples such as:
```zsh
 bindsym XF86MonBrightnessUp exec swayosd --brightness raise
 bindsym XF86AudioRaiseVolume exec swayosd --output-volume raise
```
I know that passing in '--help'  tells how to use the custom amounts for the change of brightness and volume. However, there is an example of using max volume with a value within the README:
```zsh
exec swayosd --max-volume 120
```

So I think there should be examples of using custom values for all the brightness and volume within the README. I am willing to do a pr adding to the README file. Before I begin, I am wondering if I should have a sub-section for the Sway examples with something like:

```zsh
# Brightness raise custom value
bindsym XF86MonBrightnessUp + <key> exec swayosd --brightness 10
# Brightness lower custom value
bindsym XF86MonBrightnessDown + <key> exec swayosd --brightness -10
# Volume raise custom value
bindsym XF86AudioRaiseVolume + <key> exec swayosd --output-volume 10
# Volume raise custom value
bindsym XF86AudioRaiseVolume + <key> exec swayosd --output-volume -10
```
Or should it be somewhere else?

### PR DESCRIPTION
**Pull request by trollLemon**
This pr adds to the documentation in the README file, showing how to use custom values when raising/lowering volume or brightness.
Resolves #39

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,15 +80,34 @@ bindsym XF86AudioMute exec swayosd-client --output-volume mute-toggle
...
+# Volume lower with custom value
+bindsym XF86AudioRaiseVolume + <key> exec swayosd-client --output-volume -15
+
+# Sink volume raise with custom value optionally with --device
+bindsym XF86AudioLowerVolume + <key> exec  swayosd-client --output-volume +10 --device alsa_output.pci-0000_11_00.4.analog-stereo.monitor
+# Sink volume lower with custom value optionally with --device
+bindsym XF86AudioLowerVolume + <key> exec  swayosd-client --output-volume -10 --device alsa_output.pci-0000_11_00.4.analog-stereo.monitor
+
+
+
```
Comment (ErikReider) [inline README.md:94]: Remove these extra lines please. Should be 1 empty line between each "category" of binding

File: README.md [hunk 2]
```diff
@@ -80,15 +80,34 @@ bindsym XF86AudioMute exec swayosd-client --output-volume mute-toggle
 # Source volume toggle mute
 bindsym XF86AudioMicMute exec swayosd-client --input-volume mute-toggle
 
+# Volume raise with custom value
+bindsym XF86AudioRaiseVolume + <key> exec swayosd-client --output-volume 15
```
Comment (ErikReider) [inline README.md:84]: These aren't functional without modification so removing the ` + <key>` :)

**Review by ErikReider — CHANGES_REQUESTED** [review]

**Comment by ErikReider** [comment]
Some minor nits

File: README.md (hunk 1 above)
Comment (trollLemon) [reply README.md]: Spaces removed

File: README.md (hunk 2 above)
Comment (trollLemon) [reply README.md]: Removed the + <key> from all the examples.

**Review by ErikReider — APPROVED** [review]
LGTM! Thanks :)

**Pull request merged**